en/components/tree.md

```diff
@@ -0,0 +1,314 @@
+[data]({environment:angularApiUrl}/classes/igxtreenodecomponent.html#data) property can be used to add a reference to the data entry the node represents. Binding `[data]` is required for searching through nodes using [IgxTreeComponent.findNodes()]({environment:angularApiUrl}/classes/igxtreecomponent.html#findNodes).
+
+### Declaring a tree
+
+Nodes can be declared using one of the following approaches.
+
+- Declaring the tree and its nodes by specifying the node hierarchy and iterating through a data set
+
+```html
+<igx-tree>
+	<igx-tree-node *ngFor="let node of data" [data]="node" [expanded]="isNodeExpaded(node)" [selected]="isNodeSelected(node)">
+		{{ node.text }}
+		<img [src]="node.image" alt="node.imageAlt" />
+		<igx-tree-node *ngFor="let child of node.children" [data]="child" [expanded]="isNodeExpaded(child)" [selected]="isNodeSelected(child)">
+            {{ child.text }}
+		</igx-tree-node>
+	</igx-tree-node>
+</igx-tree>
+```
+
+Nodes can be bound to a data model so that their expanded and selected states are reflected in the underlying data as well.
+
+```html
+<igx-tree (nodeSelection)="handleSelectionEvent($event)">
+	<igx-tree-node *ngFor="let nodes of data" [data]="node" [(expanded)]="node.expanded" [(selected)]="node.selected">
+		{{ node.text }}
+		<img [src]="node.image" alt="node.imageAlt" />
+		<igx-tree-node *ngFor="let child of node.children" [data]="child">
+			<a igxTreeNodeLink href="child.url" target="_blank">
+                {{ child.text }}
+            </a>
+		</igx-tree-node>
+	</igx-tree-node>
+</igx-tree>
+```
+
+- Declaring a tree by creating static unbound nodes
+
+In order to render a tree you do not necessarily need a data set - individual nodes can be created without an underlying data model:
+
+```html
+<igx-tree>
+	<igx-tree-node [expanded]="true" [selected]="false">
+		I am a parent node 1
+		<img [src]="hard_coded_src.webb" alt="Alt Text" />		
+		<igx-tree-node [expanded]="true" [selected]="false">
+			I am a child node 1
+			<igx-tree-url-node [url]="https://google.com">
+				I am a child node of the child
+			</igx-tree-url-node>
+		</igx-tree-node>
+	</igx-tree-node>
+	
+	<igx-tree-node [expanded]="false" [selected]="false">
+		I am a parent node 2
+		<img [src]="hard_coded_src.webb" alt="Alt Text" />
+        <igx-tree-node [expanded]="false" [selected]="false">
+			I am a child node 1
+		</igx-tree-node>
+	</igx-tree-node>
+	
+    <igx-tree-node [selected]="false" [disabled]="true">
+		I am a parent node 3
+	</igx-tree-node>
+</igx-tree>
+```
+
+### Nodes with links
+When a node should render a link, the `IgxTreeNodeLink` directive should be added to the <**a**> tag. This will ensure the proper aria role is assigned to the node's DOM elements.
```

Comment:
```suggestion
When a node should render a link, the `IgxTreeNodeLink` directive should be added to the `<a>` tag. This will ensure the proper aria role is assigned to the node's DOM elements.
```